doc/user/oss_backup.md

```diff
@@ -0,0 +1,125 @@
+# Backups using Alibaba Cloud Object Storage Service (OSS)
+
+Etcd backup operator backs up the data of an etcd cluster running on Kubernetes to a remote storage such as Alibaba Cloud Object Storage Service (OSS). If it is not deployed yet, please follow [here](walkthrough/backup-operator.md#deploy-etcd-backup-operator) to deploy it, e.g. by running
+
+```sh
+kubectl apply -f example/etcd-backup-operator/deployment.yaml
+```
+
+## Setup Alibaba Cloud backup account, OSS bucket, and Secret
+
+1. Login [Alibaba Cloud Console](https://www.alibabacloud.com)(or [Aliyun Console](https://www.aliyun.com/) if you are in China ), and then create your own AK/SK, you can optionally create a OSS bucket for backups.
```

Comment:
Define the correct acronyms for AccessKeyID (`AKID`) and AccessKeySecret (`AKS`) before referencing them. With this, also change `AK`=>`AKID` and `SK`=>`AKS` in the rest of the doc.
Also define and link to OSS doc.

```suggestion
1. Login [Alibaba Cloud Console](https://www.alibabacloud.com) (or [Aliyun Console](https://www.aliyun.com/) if you are in China) and create your own [AccessKey](https://www.alibabacloud.com/help/doc-detail/29009.htm) which gives you the AccessKeyID (AKID) and AccessKeySecret (AKS). You can optionally create an Object Storage Service ([OSS](https://www.alibabacloud.com/help/doc-detail/31947.htm)) bucket for backups.
```